src/lib/modules/Patron/PatronBulkExtendLoans/PatronBulkExtendLoans.js

```diff
@@ -0,0 +1,54 @@
+import { Truncate } from '@components/Truncate';
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Button, Icon, Label, Popup } from 'semantic-ui-react';
+
+export default class PatronBulkExtendLoans extends Component {
+  render() {
+    const { patronPid, bulkLoanExtension, isLoading, ...props } = this.props;
+    return (
+      <Button
+        labelPosition="right"
+        {...props}
+        primary
+        onClick={() => bulkLoanExtension(patronPid)}
+      >
+        <Button
+          labelPosition="left"
+          fluid
+          icon
+          primary
+          loading={isLoading}
+          {...props}
+        >
+          <Icon name="refresh" />
+          <Truncate>Extend all loans</Truncate>
+        </Button>
+        <Label basic className="button-with-additional-info" {...props}>
+          <Popup
+            className="button-with-additional-info"
+            trigger={
+              <Icon
+                name="question circle"
+                fitted
+                className="button-with-additional-info"
+              />
+            }
+            content="This action will have an effect only on the eligible loans."
```

Comment:
I would suggest having a more clear msg, maybe? Something like (but with better English):
```suggestion
            content="Loans will be extended only for literature for which there is no request from other users."
```